src/renderer/pages/Staking/Operations/ChangeValidators/InitOperation/InitOperation.tsx

```diff
@@ -195,41 +161,25 @@ const InitOperation = ({ api, chainId, accounts, asset, addressPrefix, onResult
             onDepositChange={setDeposit}
           />
         }
+        header={
+          <OperationHeader
+            chainId={chainId}
+            accounts={accounts}
+            isMultiselect
+            invalid={accountIsMultisig ? !isValidDeposit : !isValidFee}
```

Comment:
Looks like this are general validations. Probably we can create include them into OperationHeader or create a helper function for them. The main point not to duplicate them in each module